shared/src/components/badge.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+import { Chip } from '@material-ui/core';
+import { withStyles } from '@material-ui/core';
+import { COLORS, BASE_FONT } from '../styles';
+
+interface BadgeProps {
+  value?: string;
+}
+
+const StyledChip = withStyles({
+  root: {
+    color: COLORS.white,
+    backgroundColor: '#555555',
```

Comment:
We've been trying to support both dark and light theme modes. As such, we've tried to rely on the JupyterLab color variables as much as possible rather than fixed colors.

COLORS.white already handles this. I think we might be able to use one of the jp-inverse-layout-color variables for this perhaps.

https://github.com/jupyterlab/jupyterlab/blob/1.2.x/packages/theme-light-extension/style/variables.css

Reply:
Ah gotcha. Testing out that change!